chain/client/src/client_actor.rs

```diff
@@ -55,6 +53,8 @@ pub struct ClientActor {
     pub adv_sync_info: Option<(u64, u64)>,
     #[cfg(feature = "adversarial")]
     pub adv_disable_header_sync: bool,
+    #[cfg(feature = "adversarial")]
+    pub adv_disable_doomslug: bool,
```

Comment:
This variable appears to not be used anywyere? (it is assigned to, but I can't find it being read from)